index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>SergSkol Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="container header_flex">
+        <div class="logo">My Logo</div>
+        <div class="logo">≡</div>
```

Comment:
- [ ] It is required to wrap your hamburger icon in a `<nav>` tag for semantics reason. I would also recommend you to use font icon for your hamburger icon. You can find that [here](https://fontawesome.com/search?q=bar&o=r)

Reply:
Thank you! Done!